references/oidc-mock/test/integration/features/oidc-mock.end-to-end.feature

```diff
@@ -0,0 +1,48 @@
+@End-to-EndTests
+Feature:
+  As a Client App 
+  I want to get an access token from an identity provider
+  So that I can retrieve different types of information
+
+  Scenario: User Authorizes
+    Given I navigate to the authorize page
+    When I sign in and consent
+    Then I am redirected to the Client App
+    And I receive an auth code in a query param
+    And I store the auth code in global scope
+    And I store the state parameter in global scope
+
+  Scenario: Generate Access Token
+    Given I have basic authentication credentials `clientId` and `clientSecret`
+    And I set form parameters to 
+      | parameter   | value		                |
```

Comment:
spaces look a bit funny here - try opening them in vim and fixing them if your normal editor isnt showing them correctly